static/app/components/feedback/list/feedbackList.tsx

```diff
@@ -28,6 +28,24 @@ const cellMeasurer = {
   minHeight: 24,
 };
 
+function Message({title, subtitle}: {subtitle: React.ReactNode; title: React.ReactNode}) {
+  return (
+    <Fragment>
+      <EmptyMessage>{title}</EmptyMessage>
+      <p>{subtitle}</p>
+    </Fragment>
+  );
+}
```

Comment:
Is this necessary? It seems to just be used in `NoFeedback`, you can inline it in there.